src/NuGet.Core/NuGet.Frameworks/FrameworkConstants.cs

```diff
@@ -60,6 +60,7 @@ public static class FrameworkIdentifiers
             public const string MonoMac = "MonoMac";
             public const string XamarinIOs = "Xamarin.iOS";
             public const string XamarinMac = "Xamarin.Mac";
+            public const string XamarinMacCatalyst = "Xamarin.MacCatalyst";
```

Comment:
I don't see this being used anywhere, am I missing something?

Reply:
oh. No, this should not exist lol. This was a brain fart. :|